### DISCUSSION TITLE
Get started contributing

### DISCUSSION THREAD

**Discussion by obventio56** [AltimateAI/vscode-dbt-power-user#479] [General] [answered]
Hello,

I would like to try and contribute. Do you have any guides on setting up the extension for development? 

Thanks!
Alex

**Comment by mdesmet** [comment, marked as answer]
It just works like a typical typescript project.

1. check out the project with git
2. run `npm install`
3. Make some code changes
4. Test your changes by launching the project in debug mode
5. Point to your favourite dbt project and test

**Comment by anandgupta42** [comment]
@obventio56 We now have a contribution guideline also : https://github.com/innoverio/vscode-dbt-power-user/blob/master/CONTRIBUTING.md

Hope to see a contribution from you soon. Feel free to reach out if you have any questions. We can also be found at the [ #tools-dbt-power-user](https://getdbt.slack.com/archives/C05KPDGRMDW) channel in DBT Community slack.

-Anand
